### PR TITLE
Feature/display temperature unit

### PR DESCRIPTION
**Pull request by jr7square**

### CODE REVIEW

**Comment by jr7square** [comment]
Issue #35

File: app.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def weather(city, state):
     #print out a statement with the current weather info + location that was used/detected
     #return("Right now in "+ city + ", " + state + " it is " + temperature +  degree_sign + " and there  is a " + RAIN_WARNING)
     location = {'city': city, 'state': state}
-    weather_info = {'temperature' : temperature, 'rain' : rain_commentary}
+    weather_info = {'temperature' : temperature, 'rain' : rain_commentary, 'celsius'=config.celsius}
```
Comment (M0nica) [inline app.py]: This throws a syntax error. There should be a `:` and not `=`. How were you about to get this running?

Comment (M0nica) [inline app.py]: Also, when Celsius doesn't exist in Config (which is a setup that should be supported) the following error will be thrown:
```AttributeError: module 'config' has no attribute 'celsius'```

Are you able to write this in a way that can handle missing celsius config value?

**Comment by M0nica** [comment]
Thank you for your contribution. I had a few comments that should be addressed before this can be merged.

File: app.py (hunk 1 above)
Comment (jr7square) [reply app.py]: oh correct. Will fix

Comment (jr7square) [reply app.py]: I can fix this too. No problem

**Comment by M0nica** [comment]
Thank you for your contribution! This ended up being implemented in another PR so I am going to close this one but do appreciate you work!

**Pull request closed without merging**